python/OpenVINO_EP/azureml/bert-squad-qa-inference-using-int8-model.ipynb

```diff
@@ -0,0 +1,741 @@
+            "deleting": false
+          }
+        }
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "env = Environment(environment_name)\n",
+        "env.docker.base_image = None\n",
+        "env.docker.base_dockerfile = dockerfile\n",
+        "env.python.user_managed_dependencies = True"
+      ],
+      "outputs": [],
+      "execution_count": 8,
+      "metadata": {
+        "collapsed": false,
+        "gather": {
+          "logged": 1670306216906
+        },
+        "jupyter": {
+          "outputs_hidden": false,
+          "source_hidden": false
+        },
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Mounting Azure Storage for easier Access.\r\n",
+        "- We need access to the corresponding keys for our storage.\r\n",
+        "- The keys can be accessed from [here](https://learn.microsoft.com/en-us/azure/storage/common/storage-account-keys-manage?toc=%2Fazure%2Fstorage%2Fblobs%2Ftoc.json&tabs=azure-portal#view-account-access-keys)\r\n",
+        "- We mount the storage in the container for easy access of large files.\r\n",
+        "\r\n",
+        "## Creating job config for runnning the model\r\n",
+        "\r\n",
+        "\r\n",
+        "Defining the set of commands that will be used to perform our desired task.  \r\n",
+        "    1. Exporting some environment variables which are accessed for execution configurations during inference.  \r\n",
+        "    2. Running the inference using OpenVINOExecutionProvider"
+      ],
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      }
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Azure Credentials\r\n",
+        "Here we declare variables which will be used to connect to our Azure account. Storage account details such as storage account name and key can be accessed from :- Go to Azure portal > Storage Account >  Security + networking > Access keys. Also file share name can accessed from:- Go to corresponding file shares > Settings > Properties."
+      ],
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "# Storage account name\r\n",
+        "STORAGEACCT= \"\"\r\n",
+        "# Storage account key\r\n",
+        "STORAGEKEY = ''\r\n",
+        "# Share name\r\n",
+        "SHARE = ''\r\n",
+        "\r\n",
+        "MOUNT_PATH = \"/mnt/MyAzureFileShare\"\r\n",
+        "modelname = \"bert-large-uncased-whole-word-masking-finetuned-squad\"\r\n",
+        "\r\n",
+        "# Path to the Finetuned INT8 ONNX Model directory.\r\n",
+        "# e.g. modelpath = f\"{MOUNT_PATH}/models\"\r\n",
+        "modelpath = f\"{MOUNT_PATH}/\"\r\n"
```

Comment:
Is this where one would enter the path to the int8 model generated in the training notebook?